features.json

```diff
@@ -2,60 +2,105 @@
 	"$schema": "./features.schema.json",
 	"features": {
 		"bigInt": {
-			"description": "JS BigInt to Wasm i64 integration",
+			"description": "JavaScript BigInt to WebAssembly i64 integration",
 			"url": "https://github.com/WebAssembly/JS-BigInt-integration",
 			"phase": 5
 		},
+		"branchHinting": {
+			"description": "Branch Hinting",
+			"url": "https://github.com/WebAssembly/branch-hinting/blob/master/proposals/branch-hinting/Overview.md",
+			"phase": 5
+		},
 		"bulkMemory": {
 			"description": "Bulk memory operations",
 			"url": "https://github.com/WebAssembly/bulk-memory-operations/blob/master/proposals/bulk-memory-operations/Overview.md",
 			"phase": 5
 		},
+		"compilationHints": {
+			"description": "Compilation Hints",
+			"url": "https://github.com/WebAssembly/compilation-hints/blob/main/proposals/compilation-hints/Overview.md",
+			"phase": 1
+		},
+		"componentModel": {
+			"description": "Component Model",
+			"url": "https://github.com/WebAssembly/component-model",
+			"phase": 1
+		},
+		"customAnnotationSyntaxInTheTextFormat": {
+			"description": "Custom Annotation Syntax in the Text Format",
+			"url": "https://github.com/WebAssembly/annotations/blob/main/proposals/annotations/Overview.md",
+			"phase": 5
+		},
 		"customPageSizes": {
-			"description": "Custom page sizes",
+			"description": "Custom Page Sizes",
 			"url": "https://github.com/WebAssembly/custom-page-sizes/blob/main/proposals/custom-page-sizes/Overview.md",
 			"phase": 1
 		},
+		"esmIntegration": {
+			"description": "ESM integration",
+			"url": "https://github.com/WebAssembly/esm-integration",
+			"phase": 3
+		},
```

Comment:
This is the only one that's not implemented by any, but is in phase 3, which falls into my "everything phase 3 and up having to be listed" rule.

Reply:
Nvm, turns out it's implemented by Webkit.

Reply:
Interesting, I wasn't aware of this. Do you know if it's up-to-date? There have been quite a few changes in the long time since the proposal started. Maybe @guybedford knows.
If it's the original version of the proposal and not the current one, we probably shouldn't include it.

Reply:
I'd guess it's not up to date, but I don't actually know.